bothan-binance/src/worker/builder.rs

```diff
@@ -83,13 +82,3 @@ impl BinanceWorkerBuilder {
         Ok(worker)
     }
 }
```

Comment:
why delete default na

Reply:
store doesn't have default anymore due to new internal requirements so store now needs to be passed in, hence the removal of default of anything downstream